hooks/post-command

```diff
@@ -11,13 +11,24 @@ if [[ "${#CORE_DUMPS[@]}" > 0 ]]; then
     echo "--- List of core dumps"
     ls -lah "${CORE_DUMPS[@]}"
 
-    # Start compressing/collecting corefiles in the background
+    # Start a subshell for compressing/collecting corefiles in the background in parallel
+    (
     PIDS=()
     for COREFILE in "${CORE_DUMPS[@]}"; do
-        compress_bundle "${COREFILE}" &
+        (
+        compress_bundle "${COREFILE}"
+        buildkite-agent artifact upload "$(compressed_bundle_names "${COREFILE}")"
+        ) &
         PIDS+=( "$!" )
     done
 
+    # Wait for compression and upload to finish
+    echo "--- Waiting upon compression/bundling"
```

Comment:
This can now interfere with the `echo "--- ${COREFILE} -> ${CMD}"` on line 39, as seen in [this build](https://buildkite.com/julialang/coreupload-buildkite-plugin/builds/148#01904cce-83b0-4433-a411-73be86868585/86-89) where the backtrace is printed in the `Waiting upon compression/bundling` section, instead of in the `foo/bash-pid89513.core -> bt` section.

Reply:
Oops, yes, I should moved this above before the subshell was spawned